.github/workflows/dist-docs.yml

```diff
@@ -0,0 +1,127 @@
+name: Rebuild src/static/support/dist-docs
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build-linux:
+    env:
+      dependencies: |
+        automake libtool gcc bc libjemalloc1 libjemalloc-dev    \
+        libssl-dev llvm-dev libelf-dev libnuma-dev libpcap-dev  \
+        selinux-policy-dev \
+        ghostscript markdown
+      m32_dependecies: gcc-multilib
+      CC:          ${{ matrix.compiler }}
+      LIBS:        ${{ matrix.libs }}
+      M32:         ${{ matrix.m32 }}
+      OPTS:        ${{ matrix.opts }}
+      TESTSUITE:   ${{ matrix.testsuite }}
+      ASAN:        ${{ matrix.asan }}
+
+    name: linux ${{ join(matrix.*, ' ') }}
+    runs-on: ubuntu-18.04
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - compiler:     gcc
+            opts:         --disable-ssl
+
+    steps:
+    - name: checkout ovn-website
+      uses: actions/checkout@v2
+      with:
+        path: 'ovn-website'
+        ref: 'master'
+
+    - name: checkout OVN master
+      uses: actions/checkout@v2
+      with:
+        repository: 'ovn-org/ovn'
+        path: 'ovn'
+        ref: 'master'
+        submodules: recursive
+
+    - name: update APT cache
+      run:  sudo apt update
+
+    - name: install required dependencies
+      run:  sudo apt install -y ${{ env.dependencies }}
+
+    - name: install libunbound libunwind
+      if:   matrix.m32 == ''
+      run:  sudo apt install -y libunbound-dev libunwind-dev
+
+    - name: install 32-bit dependencies
+      if:   matrix.m32 != ''
+      run:  sudo apt install -y ${{ env.m32_dependecies }}
+
+    - name: update PATH
+      run:  |
+        echo "$HOME/bin"        >> $GITHUB_PATH
+        echo "$HOME/.local/bin" >> $GITHUB_PATH
+
+    - name: set up python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+
+    - name: prepare ovn
+      run:  |
+        pushd ./ovn
+        ./.ci/linux-prepare.sh
+        popd
+
+    - name: build ovn
+      run:  |
+        pushd ./ovn
+        ./.ci/linux-build.sh
+        popd
+
+    - name: build ovn dist-docs
+      run:  |
+        pushd ./ovn
+        make dist-docs
+        popd
```

Comment:
It's a shame that you have to build OVN completely even though all you really want are the dist-docs. It might be a good idea for us to expand OVN's linux-build.sh script to accept `make` targets so that after it configures OVS and OVN, it just makes the specific target. That's not something you can address in this PR, but it's just something I thought about.

Reply:
> It's a shame that you have to build OVN completely even though all you really want are the dist-docs. It might be a good idea for us to expand OVN's linux-build.sh script to accept `make` targets so that after it configures OVS and OVN, it just makes the specific target. That's not something you can address in this PR, but it's just something I thought about.

yes! I did not want to bite that off in this PR, as you pointed out.
One "pro" in doing that build is that we kinda ensure that the docs are being built on top of a 'solid' branch/tag. Since this dist-docs build should be something we do seldomly, I find the extra time to hopefully not be a problem. But I reserve the right to be wrong on that, ok? :)